lib/cc/analyzer/issue_relative_path_validation.rb

```diff
@@ -11,6 +11,13 @@ def message
 
       private
 
+      def relative_to?(directory)
```

Comment:
`relative_path_from` doesn't behave very well in terms of trying to resolve a relative path in terms of an absolute path: it errors out. I don't think that's really what it's intended for anyway, since it doesn't actually touch the file system, it's just manipulating the path independent of the real FS.

`#cleanpath` may be useful: you could check that `path == Pathname.new(path).cleanpath.to_s`.